lib/train/extras/os_common.rb

```diff
@@ -83,58 +87,46 @@ def to_hash
 
     private
 
+    # TODO: extend base implementation for detecting the family type
+    # to Windows and others
     def detect_family
-      # if some information is already defined, try to verify it
-      # with the remaining detection
-      unless @platform[:family].nil?
-        # return ok if the preconfigured family yielded a good result
-        return true if detect_family_type
-        # if not, reset the platform to presets and run the full detection
-        # TODO: print an error message in this case, as the instantiating
-        # backend is doing something wrong
-        @platform = {}
-      end
-
-      # TODO: extend base implementation for detecting the family type
-      # to Windows and others
+      # may be initialized by connection
+      return @platform[:family] unless @platform[:family].nil?
       case uname_s
       when /unrecognized command verb/
```

Comment:
`/unrecognized command verb/` could be omitted if you want and it would just fall under the `else` case.